main.py

```
@@ -11,23 +11,69 @@
 from config_manager import ConfigManager
 from crawler import WebCrawler
 from converter import MarkdownConverter
+from error_types import ErrorHandler, FileSystemError, ErrorSeverity
+from logging_manager import setup_logging, LoggingManager, StructuredLogger
```

Ruff (0.11.9)

`logging_manager.LoggingManager` imported but unused

Remove unused import: `logging_manager.LoggingManager`

(F401)